### DISCUSSION TITLE
How to contribute to translate in korean?

### DISCUSSION THREAD

**Discussion by Jeffr-K** [prisma/prisma#11200] [Q&A] [answered]
Hello, All of prisma user.

I am a newbie prisma user. I want to contribute to translate for korean.
Can I do it now? is there any guide for contributing?

Thank you all :)

**Comment by tanberry** [comment, marked as answer]
HI there, @IntoTheDeepSea! Welcome to the Prisma community; we are so happy to have all newbies and hardened veterans. ;-)

Interesting offer to translate the Prisma docs into Korean, thank you, but we won't be able to host those docs on our side for a few reasons. Mostly that we do not have the capability to test them and verify the contents before publishing. Also, very importantly, the Prisma docs are somewhat volatile, and require constant refactoring and adding new content, as we release new features and refactor code... so any translations would have to be as constantly updated. And we do not have an external mechanism, or channel for communication, that is sufficient for the amount of changes.

However, if you are interested in hosting a Korean-language set of docs you can do so, just please refrain from using our logos or anything that might make it appear like it is an official Prisma web site. ;-)

We do have a [Contributor's Guide](https://github.com/prisma/docs/blob/main/CONTRIBUTING.md), and we would very much appreciate contributions to our docs! Anything from tiny typos to rewording steps to creating an entire tutorial, we welcome contributions. Thank you in advance!

**Reply by Jeffr-K** [reply]
Thank you for guiding. I think more about it. :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
